### DISCUSSION TITLE
Attach PGP public key to outgoing SimpleLogin emails

### DISCUSSION THREAD

**Discussion by MementoRebel** [simple-login/app#2106] [Feature Requests]
Currently there is no way to initiate an encrypted PGP email conversation using SimpleLogin aliases. You can encrypt the first outgoing email by adding your contact's PGP key to the SimpleLogin reverse alias but that contact will not be able to reply with encryption as SimpleLogin cannot/does not currently attach any public key information for aliases.

**Comment by hashara256** [comment]
I believe you should attach your own public key. 
U can reuse 1 keypair for as many aliasses as u like, however be carefull to remove any identifiers and replace your main adress with the alias.
